users/engler/fairseq/training.py

```diff
@@ -86,6 +98,7 @@ def __init__(
             (usually in the same folder as python exe '.../bin/')
         :param Path|str fairseq_root: File path to the fairseq git for alternative call of fairseq-hydra-train
             (no need to install fairseq here)
+        :param bool cached: enables caching of the data given in the manifest
```

Comment:
Maybe make it explicit that the i6 cache manager is used. The variable could also be named `use_cache_manager` similar to RETURNN.